app/views/documents/history/_whitehall_entry.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="app-timeline-entry">
+  <% if entry.details.imported_from_whitehall? %>
+    <%= render "govuk_publishing_components/components/notice", {
+      title: "#{t "documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}"}",
```

Comment:
This is a little fiddly to read. Can I suggest...?

```suggestion
      title: "#{t("documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}")}",
```

Reply:
Done - although I did it a slightly different way.